src/docs/installation.md

```diff
@@ -30,6 +30,8 @@ It allows you to setup a fresh Vue Native app with either Expo or React Native C
 
 Expo was designed to allow developrs to quickly set up and develop React Native apps, without having to configure Xcode or Android Studio. It is most suitable for you if you come from a Web background.
 
+*Before starting the installation you may need to install [Yarn](https://yarnpkg.com/lang/en/):*
```

Comment:
yarn is not necessary for installation. In case you encounter errors during installation without yarn, please create an issue on the [vue-native-cli repo](https://github.com/GeekyAnts/vue-native-cli/issues)